Resources/Prototypes/Entities/Clothing/Head/eva-helmets.yml

```
@@ -52,15 +52,17 @@
 
 #Paramedic Void Helmet
 - type: entity
-  parent: ClothingHeadEVAHelmetBase
+  parent: ClothingHeadHardsuitWithLightBase # goobstation - why didnt this have a light
   id: ClothingHeadHelmetVoidParamed
   name: paramedic void helmet
   description: A void helmet made for paramedics.
   components:
   - type: Sprite
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)